### ISSUE TITLE
Copying text messes with the formatting

### ISSUE THREAD

**Issue by Victor239** [Mosc/Glider#24]
When trying to copy comments to the clipboard e.g. https://news.ycombinator.com/item?id=27558698, it messes with the URL and adds `<p>` tags:

```
Governments around the world have already started to pilfer data collected from these apps, as was predictable. [1]<p>[1] <a href="https:&#x2F;&#x2F;www.9news.com.au&#x2F;national&#x2F;wa-police-stand-by-decision-to-use-safewa-contact-tracing-app-in-nick-martin-murder-investigation&#x2F;9250eb64-b523-46ea-810d-bf09521f1e1f" rel="nofollow">https:&#x2F;&#x2F;www.9news.com.au&#x2F;national&#x2F;wa-police-stand-by-decisio...</a>
```

**Comment by Victor239** [comment]
Also happens in normal text blocks actually. https://news.ycombinator.com/item?id=27580416

```
Quite different. 
ZT is centralized config, direct peer to peer, automatic key exchange, virtual switch.<p>YG is decentralized, direct peer to peer and multi hop routing, manual key exchange for direct peering, virtual internet (each node is a router to another nodes)<p>ZT (and Tailscale) requires a central node coordinator for automatic config and peer key exchanges, peers directly to each node to create a flat virtual network, not multi hop routing, between two peers you need a reachable IP or port mapping (supports UPNP) by one of them (fixed in config there are private TCP relays when carrier NAT&#x2F;double NAT&#x2F; restrictive firewalls are in place, very slow), Uses UDP for the links, mimics a virtual switch and support custom IPV4 &#x2F; IPV6, good for virtual private networks.<p>YG (and CJDNS) is kinda an overlay public network, is fully decentralized, it supports automatic routing between nodes to mimic a virtual Internet, each node is a router too, needs to register each key for every direct peer node connection (config needs peer key, reachable ip and port, but supports automatic key exchange for local networks), YG uses TCP for the links, support self-healing routing, every peer has an IPV6 address derived by its public key.<p>ZT (and Tailscale) can mimic a mesh network and node routing because supports bridge nodes between networks (routing between virtual switches), but is not self-healing and somewhat heavy work to config. ZT is fully open source, can be config with your own node coordinator and discovery helpers (Controllers and Moons in ZT), not easy. Only Taislcale client is open source, can&#x27;t be config with your own node coordinators for free.<p>You can mix ZT and YG, weird side effects warranted but works.
(Edited post - some grammar fixes)
```

**Comment by Mosc** [comment]
Yeah, Glider is a bit lazy in this regard. The raw HTML is copied rather than the text, the latter usually being what one wants. Planning to fix that. This fix is not always ideal though. Consider your first link. It would be copied as "https://www.9news.com.au/national/wa-police-stand-by-decisio..." rather than the full URL because that is how it is displayed, rendering it pretty useless. Working around that would involve some more involved parsing.

**Comment by Mosc** [comment]
Fixed in 790838b6b6f013f37afdf27a25ebbf00b41bdc98.
